jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/RadioLabelTests.java

```diff
@@ -0,0 +1,61 @@
+package io.github.epam.html.tests.elements.complex;
+
+import io.github.epam.TestsInit;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static io.github.com.StaticSite.*;
+import static io.github.com.pages.MetalAndColorsPage.*;
+import static io.github.epam.html.tests.site.steps.States.*;
+import static java.util.Arrays.*;
+import static org.hamcrest.Matchers.*;
+import static org.testng.Assert.*;
+
+public class RadioLabelTests implements TestsInit {
+
+    @BeforeMethod
+    public void before() {
+        shouldBeLoggedIn();
+        metalAndColorsPage.shouldBeOpened();
+        odds.select(text);
+    }
+    String text = "5";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5511453)